src/About/style.scss

```diff
@@ -0,0 +1,96 @@
+@import './_variables.scss';
+
+.wrapper {
```

Comment:
if class wrapper is used in a few places and its styles are the same it should be replaced in general.scss. Otherwise pls change the name to more appropriate

Reply:
i have changed the name of wrapper